src/schema/v2/SearchableItem/SearchableItemPresenter.ts

```diff
@@ -34,6 +34,7 @@ export class SearchableItemPresenter {
         return `Browse current exhibitions in ${display}`
       case "Collection":
       case "ArtistSeries":
+      case "ViewingRoom":
```

Comment:
@mzikherman I got lost in a maze of Gravity dataloaders trying to understand where `description` (see line 16 of this file) and `imageUrl` come from. Any pointers for me?

I don't think that blocks merging this PR because everything we need for autosuggest already works, but we'll want those things eventually for full search page!

Reply:
search results (autosuggest and search), have custom JSON returned for them: https://github.com/artsy/gravity/blob/b94d4f004a99cb0100358b6ce438c76d0894691a/app/api/util/search_helpers.rb#L48 .

This is opposed to filter endpoints, where ElasticSearch is responsible for just returning id's, and then we look those up in our database and return the 'normal' JSON for those objects. In the search endpoints, we actually return entire documents from ElasticSearch and return those directly, w/o hydrating from our database.

Essentially, if you index a field called `description` for viewing rooms, that field will be returned in the search results JSON automatically, and should flow through here. Similarly for `image_url`.

Reply:
That's helpful, thank you! I'll touch base with product & design about what we want to use for those fields and follow up with a Gravity PR. For now though, sounds like that doesn't get in the way of merging this PR and shouldn't require any future MP updates, just Gravity. @jpotts244 if everything looks reasonable to you, want to go ahead and merge?